packages/core/src/request.ts

```diff
@@ -45,6 +45,25 @@ export function sessionToSentryRequest(session: Session, api: API): SentryReques
   };
 }
 
+/** Creates a SentryRequest from an Aggregates of Request mode sessions */
```

Comment:
This whole function is an exact copy of `sessionToSentryRequest`. Can we unify it? The only difference is `type`, which can be provided argument.